### PR TITLE
GUI Buttons for the start and Game over screens.

### PR DESCRIPTION
**Pull request by Ashthecool**
- Made GUI Buttons for Start screen and Game Over screens.
- Made when they are hovered change texture.

*Find problems or ideas*

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -338,22 +361,10 @@ def on_draw(self):
             center_y=CONFIG['TITLE_Y']
         )
 
-        self.play_button.draw_scaled(
-            center_x=CONFIG['BUTTON_X'],
-            center_y=CONFIG['BUTTON_Y'],
-        )
-
-    def on_key_press(self, symbol: int, modifiers: int):
```
Comment (tobiasbp) [inline my_game.py:346]: Hey! Don't remove the press key to play feature :(

File: my_game.py [hunk 2]
```diff
@@ -830,18 +861,7 @@ def on_draw(self):
             arcade.color.WHITE
         )
 
-    def on_key_press(self, symbol: int, modifiers: int):
-        self.new_game()
```
Comment (tobiasbp) [inline my_game.py:834]: Don remove the press any key feature to start feature.

**Review by tobiasbp — CHANGES_REQUESTED** [review]

**Comment by TheRiddler4K** [comment]
I got the idea that if we made a basic texture that all buttons use and then used the gui module to write text on top of it, it would be much easier to add more buttons AND these buttons could be used for the SettingsView buttons too

**Review by tobiasbp — APPROVED** [review]
Good addition. Thanks.

**Pull request merged**